python/sdist/custom_commands.py

```diff
@@ -139,6 +155,25 @@ def run(self):
 class my_build_ext(build_ext):
     """Custom build_ext to allow keeping otherwise temporary static libs"""
 
+    def build_extension(self, ext):
+        # do replacements here
+        compilerType = self.compiler.compiler_type
+        # this requires more work
+        #try: ext.extra_compile_args = ext.extra_compile_args, ext.(extra_compile_args_+compilerType)]
+        #except KeyError: None
+        if compilerType == 'msvc':
+            ECA = ext.extra_compile_args
+            for index in range(len(ECA)):
+                ECA[index] = ECA[index].replace('-std=c++11', '/std:c++17')
+            ext.extra_compile_args = ECA
+        print ('***** Extension extra_compile_args', ext.extra_compile_args)
```

Comment:
remove print statements or use logger from amici.logging.

Reply:
I have ubiquitinated the 2 print statements, so they will be degraded shortly (i.e. soon).